postgres/user_service_test.go

```diff
@@ -45,23 +45,23 @@ func TestUserService(t *testing.T) {
 			},
 			{
 				name:        tests.Faker.Name(),
-				email:       nil,
+				email:       core.EmailAddress{},
```

Comment:
dit zou bv nooit mogen kunnen

Reply:
Idd niet op de manier hoe wij die gebruiken, al kan het geen kwaad om die test te laten staan I guess?

Reply:
neen, mee eens hoor, ik heb hem daarom ook niet geschrapt

Reply:
Ik denk niet dat het veel zin heeft om dit te doen? Om deze test te doen slagen moet je nu een domeinobject dat er geldig uitziet toch nog eens expliciet checken binnen die service functie, terwijl het net de bedoeling is van die types dat dat niet nodig is 😛 het voordeel van de domeinobjecten zoals we die nu gebruiken is dat je deze testen niet meer nodig hebt omdat die cases onmogelijk zouden moeten zijn. Die test bestond omdat een interface altijd `nil` kan zijn, maar nu dat een echte struct is kan dat niet meer dus heeft het ook niet veel zin om een ongeldige versie van die struct te testen